gst/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -395,13 +401,65 @@ bb_setOption (GstTensorDec * self, int opNum, const gchar * param)
   return TRUE;
 }
 
+/**
+ * @brief check the num_tensors is valid
+*/
+inline gboolean
+_check_tensors (const GstTensorsConfig * config, const int limit)
```

Comment:
uh... is this C89-complying? (inline)

Reply:
> The inline keyword is not part of the original ANSI C standard (C89) so the library does not export any inline function definitions by default. Inline functions were introduced officially in the newer C99 standard but most C89 compilers have also included inline as an extension for a long time.
- from [GNU](https://www.gnu.org/software/gsl/manual/html_node/Inline-functions.html)

It's not for C89. I'll change it.

Reply:
add static.

Reply:
it's applied :)